portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -410,6 +412,74 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	}
 </script>
 
+<script data-senna-track="temporary" type="text/javascript">
+
+	var dropdown = document.getElementById("<%= namespace %>");
+	var button = dropdown.querySelector('.dropdown-toggle');
+	var list = dropdown.querySelector('.dropdown-menu');
+	var options = list.querySelectorAll('.dropdown-item');
+	var selectedItemValue = button.dataset.attribute;
+
+	function onButtonKeyDown(event) {
+		if (event.key === 'ArrowDown' || event.key === 'ArrowUp' || event.key === 'Enter' || event.key === 'Space') {
+			event.preventDefault();
+			list.classList.add('show');
+			button.setAttribute('aria-expanded', 'true');
+			var selectedOption = null;
+
+			Array.from(options).find(function(option) {
+				if (option.id == selectedItemValue) {
+					selectedOption = option;
+				}
+			});
+
+			if (selectedOption) {
+				selectedOption.focus();
+			}
+		}
+	}
+
+	button.addEventListener('keydown',onButtonKeyDown );
```

Comment:
The space is in the wrong spot. Should be after the comma.

Reply:
Fixed

Reply:
This still looks wrong to me: `button.addEventListener('keydown',onButtonKeyDown );` should be `button.addEventListener('keydown', onButtonKeyDown);`